repo/packages/L/linkerd/8/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "3.0",
+  "name": "linkerd",
+  "version": "0.8.3-0.1",
```

Comment:
cool. atm we're letting package maintainers decide how they want to manage it. I just want to understand. You may keep as it is.